wiki/Tools/SGL_Tool/en.md

```diff
@@ -1,11 +1,16 @@
+SGL Tool
+===========
+
 [*The new SGL Tool (Beta) - Generate Storyboards* by: MoonShade](https://osu.ppy.sh/forum/t/118733)
 
-Peppy asked me/encouraged me a long long while ago to built SGL with C\# so it can be built in osu! once it is somewhat usable in a productive environment. In this time, I almost finished the project, then lost interest and had lot of other things on the go and long story short, I now want this project to move forward again. This project is somewhat related to my old SGL, but I built everything from scratch, changed the language a lot and used other technologies to built this tool, so I think a new topic for this is needed.
+peppy asked me/encouraged me a long long while ago to built SGL with C\# so it can be built in osu! once it is somewhat usable in a productive environment. In this time, I almost finished the project, then lost interest and had lot of other things on the go and long story short, I now want this project to move forward again. This project is somewhat related to my old SGL, but I built everything from scratch, changed the language a lot and used other technologies to built this tool, so I think a new topic for this is needed.
 
 What is SGL?
```

Comment:
if the above is quoted, then remove this heading... its contents will be the lead section